docs/cis_self_assessment.md

```diff
@@ -2157,7 +2157,7 @@ Ensure that the cluster-admin role is only used where required (Not Scored)
 Inappropriate access to secrets stored within the Kubernetes cluster can allow for an attacker to gain additional access to the Kubernetes cluster or external resources whose credentials are stored as secrets.
 </details>
 
-**Result:** Pass
+**Result:** Not Scored
 
 **Remediation:**
```

Comment:
A few things:
1. The title for 5.1.2 is wrong. Looks like i copy/pasted in the title from 5.1.1. It shoiuld be:
```
Minimize access to secrets (Not Scored)
```
2. I cannot comment on it bc its too far up, but **5.1.1** needs a remediation. Just put something simple, like "RKE2 does not make inappropriate use of the cluster-admin role. Operators must audit their workloads of additional usage. See the CIS Benchmark guide for more details."
3. This (**5.1.2**) needas remediation too. How about "RKE2 limits its use of secrets for the system components appropriately, but operators must audit the use of secrets by their workloads. See the CIS Benchmark guide for more details."